test/src/chromiumonly.spec.ts

```diff
@@ -44,45 +44,33 @@ describe('Chromium-Specific Launcher tests', function () {
       }
     });
     it('should throw when using both browserWSEndpoint and browserURL', async () => {
-      const {browser, close, puppeteer} = await launch({
-        args: ['--remote-debugging-port=21222'],
-      });
-      try {
-        const browserURL = 'http://127.0.0.1:21222';
+      const {puppeteer} = await getTestState({skipLaunch: true});
+      const browserURL = 'http://127.0.0.1:21222';
 
-        let error!: Error;
-        await puppeteer
-          .connect({
-            browserURL,
-            browserWSEndpoint: browser.wsEndpoint(),
-          })
-          .catch(error_ => {
-            return (error = error_);
-          });
-        expect(error.message).toContain(
-          'Exactly one of browserWSEndpoint, browserURL or transport',
-        );
-      } finally {
-        await close();
-      }
+      let error!: Error;
+      await puppeteer
+        .connect({
+          browserURL,
+          browserWSEndpoint: 'ws://127.0.0.1:21222/devtools/browser/',
+        })
+        .catch(error_ => {
+          return (error = error_);
+        });
+      expect(error.message).toContain(
+        'Exactly one of browserWSEndpoint, browserURL or transport',
+      );
     });
     it('should throw when trying to connect to non-existing browser', async () => {
-      const {close, puppeteer} = await launch({
-        args: ['--remote-debugging-port=21222'],
-      });
-      try {
-        const browserURL = 'http://127.0.0.1:32333';
+      const {puppeteer} = await getTestState({skipLaunch: true});
```

Comment:
I suspect that test might be testing that the browser listens on the right port as well (although that is not reflected in the test name)

Reply:
I think that in not a useful test, considering that is a browser behavior it should live with the browser.
And also even if it test that it doesn't mean we can test every single port to verify that the browser listens no non expect the specified (that sounds out of scope for Puppeteer).